src/styles/index.scss

```diff
@@ -68,6 +68,10 @@
     background-color: $warning-color;
   }
 
+  .bg-ghost-white {
+    background-color: $ghost-white;
```

Comment:
We want to avoid using SCSS where possible. If you want to define a custom color, you can do the same with tailwind in the tailwind configuration file. Please do it there.